pages/Product/components/Reviews/components/AllReviewsLink/index.jsx

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2017, Shopgate, Inc. All rights reserved.
+ *
+ * This source code is licensed under the Apache 2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import I18n from '@shopgate/pwa-common/components/I18n';
+import Link from '@shopgate/pwa-common/components/Router/components/Link';
+import connect from './connector';
+import style from './style';
+
+/**
+ * Link to all reviews
+ * @param {number} totalReviewCount The number of reviews given
+ * @returns {*}
+ * @constructor
+ */
+const AllReviewsLink = ({ totalReviewCount }) => {
+  if (totalReviewCount < 2) {
```

Comment:
Please change this limiting value to be a constant.

Reply:
Done.